### ISSUE TITLE
3d image not working in full screen

### ISSUE THREAD

**Issue by my3dviews** [gkv311/sview#92]
I use sview to display 3d images in the interlaced mode on my 3d passive TV.  It worked properly until I just updated to the new 2020 version.  Now the image is split into three sections, where the top and bottom are not showing left and right correctly compared to the centre third of the image.  This problem occurs with both images and video.  It works okay when not displaying in the full screen mode where there is a border around it, but when clicking the icon in the bottom right corner, the problem shows up.  I restarted my computer and the problem persists.  I will try to see if I can go back to the previous version, as this version is not useful to me. 
Hopefully the problem can be fixed.  Thanks.

**Comment by my3dviews** [comment]
Hi: I took a photo throught the left and right lenses of my 3d glasses and are attaching them to this email.  You can see that the vertical lines are broken and that the image is split into 3 sections.     I am looking at parallel pairs with sview  displaying in the interlaced mode on my 4k passive 3d TV.    The problem even occurs with your image that is shown when the image viewer opens.  I switched it to anaglyph mode and the problem goes away.  It is only with the row interlaced in both video and image viewers.  May be due to it trying to display an extra line instead of 2160.    You would need a similar display to see the effect.  It may only be a problem on 2160p displays.  I have no way to test in on 1080p.

The problem only occurs when I go to the full screen mode (which is what I normally use), not in the mode where there is a border around it.  It never happened until I downloaded the new version.  Restarting my computer didn’t help.  I went back to the previous version, as I still had the file in my computer and it works properly.  I then reinstalled the new version, and the problem came back.

I also have windows 10 in case that helps.  My computer is set to display 3840 x 2160.

One other issue that I have had (at least in the previous version), is that when I go to full screen sometimes the mouse pointer disappears.  I can still move it around and find the buttons to click on, but that is hard to do.  If I find the full screen button at the bottom right corner and click it a couple of times, sometimes the mouse pointer comes back.  It only sometimes disappears in the full screen mode (with no border around the picture).

I hadn’t mentioned this issue before as I thought that it may just be with my computer.

One other question.   I have a Sony 3d camcorder model HDR-TD10, I believe it puts out a left and right 1080p video simultaneously.  Is there anyway to display 3d video from it with sview?  When I play the files I only get a 2d image.  The only way that I can display it in 3d is directly from the camcorder to the TV.

Thanks for replying back so soon.   I really like sview and use it all the time.





Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by gkv311** [comment]
I don't see any images in your message - probably images could be attached only via github interface.
Please don't put all issues into one thread - it would be more readable if there will be a dedicated issue per question.

> The problem only occurs when I go to the full screen mode 

Do you mean that there is no such issue even if window is resized to fullscreen (without going into fullscreen)?
Which previous sView version you are comparing to?
What is your graphic card?

**Comment by my3dviews** [comment]
![Left image](https://user-images.githubusercontent.com/70103407/91013558-1604cf80-e5a5-11ea-8066-5762775f795d.jpg)
![Right image](https://user-images.githubusercontent.com/70103407/91013559-1735fc80-e5a5-11ea-8298-9a1a3318e74e.jpg)

I am attaching the same two images directly on github.  Hopefully that works.  

Yes, I can resize the window and it works properly.  It only has the problem when going to full screen after clicking the icon in the bottom right corner, when it displays no border.  Without doing that it works no matter what size I make it, even the maximum.    

My graphics card is an NVIDIA GeForce GT 630  

The previous version that I had on my computer and works properly is file:  sViewSetup_v.17.10_23

**Comment by gkv311** [comment]
Which sView output do you use? "Row Interlaced" or something else?
Have you tried to check if option "Use texture mask (compatibility)" makes any difference?

**Comment by my3dviews** [comment]
Yes, I am using row interlaced to display on my 4k passive 3d TV using 3840 x 2160 output from my graphics card.  As far as I know the only way to display 3d with the graphics card at that resolution, other than anaglyph.  It does work in that mode, but of course it is not what I want with a 3d TV.  

I just tried the texture mask compatibility and it made no difference.  Still split into three sections.

**Comment by gkv311** [comment]
This looks just like a bug in NVIDIA drivers - have no idea how this could happen otherwise.
There is no any issues with 4K interlaced output on my system.

It might be useful, though, to check sView versions in-between currently tested releases and see in which one the issue could be first reproduced:
http://download.sview.ru/win/

**Comment by my3dviews** [comment]
I tried  sViewSetup_v.20.08_1     sViewSetup_v.20.07_14  and   sViewSetup_v.20.04_3   all had the problem.   Then tried  sViewSetup_v.19.02_10_alpha  (the one after the one that works) and it too had the problem.  Went back to sViewSetup_v.17.10_23  and it once again works properly.



Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by my3dviews** [comment]
I tried  sViewSetup_v.20.08_1     sViewSetup_v.20.07_14  and   sViewSetup_v.20.04_3   all had the problem.   Then tried  sViewSetup_v.19.02_10_alpha  (the one after the one that works) and it too had the problem.  Went back to sViewSetup_v.17.10_23  and it once again works properly.

**Comment by Fazarel** [comment]
Same problem with video, working fine except in full screen mode.
Ubuntu 20.04, sView 22.01, GPU Intel Iris Xe

**Comment by davidmachadosf** [comment]
> I use sview to display 3d images in the interlaced mode on my 3d passive TV. It worked properly until I just updated to the new 2020 version. Now the image is split into three sections, where the top and bottom are not showing left and right correctly compared to the centre third of the image. This problem occurs with both images and video. It works okay when not displaying in the full screen mode where there is a border around it, but when clicking the icon in the bottom right corner, the problem shows up. I restarted my computer and the problem persists. I will try to see if I can go back to the previous version, as this version is not useful to me. Hopefully the problem can be fixed. Thanks.



> I use sview to display 3d images in the interlaced mode on my 3d passive TV. It worked properly until I just updated to the new 2020 version. Now the image is split into three sections, where the top and bottom are not showing left and right correctly compared to the centre third of the image. This problem occurs with both images and video. It works okay when not displaying in the full screen mode where there is a border around it, but when clicking the icon in the bottom right corner, the problem shows up. I restarted my computer and the problem persists. I will try to see if I can go back to the previous version, as this version is not useful to me. Hopefully the problem can be fixed. Thanks.

Will not works. Interlaced 3d images only works when viewed on the correct resolution. The 3d display has diferent polarization on alternated lines, that split the image sended to your eyes. If you zoom or shrink the image, the correct lines information losse the information, You see only 2 messed images...

**Comment by my3dviews** [comment]
I'm well aware how 3d works on my TV.  The problem disappeared when I went back to the previous version of S-View.  It is only the later version that the problem show up.  So, I am stuck with the previous version in order for the 3d to work on my 3d TV.  I have two computers and the same issue on both.  I also have two passive 4k TVs.  Same issue on both with either computer.  The only change was the update to S-View that made it not work, then work again when I reverted back.

**Comment by davidmachadosf** [comment]
> I'm well aware how 3d works on my TV. The problem disappeared when I went back to the previous version of S-View. It is only the later version that the problem show up. So, I am stuck with the previous version in order for the 3d to work on my 3d TV. I have two computers and the same issue on both. I also have two passive 4k TVs. Same issue on both with either computer. The only change was the update to S-View that made it not work, then work again when I reverted back.

if line polarization to each eye is awaped, the efect dont works too...

**Comment by davidmachadosf** [comment]
try generate a side by side image, and test if your tv internal converter works properly

**Comment by my3dviews** [comment]
The problem wasn't that the polarization was swapped, but that the image was split as shown in the photos with either of my TVs, so not the problem of the TV.  You can always invert the polarization in SView if backwards to correct the image. 
As for side by side.  I use that feature all the time with no problems in videos and photos.  I make stereo pairs all the time that are side by side. 
Again, it is only the updated version of S-View where the problem shows up.  Now that I am running the previous version, there is no issues.
As I stated in the update of my last comment.  I have not tried any newer versions to see if they also have the issue, but I will have to leave that for a time when I have enough time to try that out.  For now, the older version works for me, so no need to update. 
Since everyone is not having the same issue, it means a compatibility issue of that version of SView with my systems.
